ledfx/api/integration_qlc.py

```diff
@@ -37,10 +37,9 @@ async def get(self, integration_id) -> web.Response:
 
         response = {}
 
-        # generate dict of {effect_id: effect_name}
-        effect_names = []
+        effect_ids = []
         for effect_type, effect in self._ledfx.effects.classes().items():
-            effect_names.append(effect.NAME)
+            effect_ids.append(effect_type)
```

Comment:
are you testing on this interface?

I appreciate the shift is subtle and looks more "correct", effect_type is equivalent to id here ( I had to go look at the code ) , and id is more definitive than name.

I just worry about side effects,

Reply:
I see you have matched ocde on the listeners, but I have no experience of the integration itself, are we confident we are not mutating the external behavior, or are you developing that hand in hand.

Reply:
Good point. I am not using this integration or performed any tests. 

This looks like it would be a breaking change in the frontend, which references the `effect_names` field. 
https://github.com/YeonV/LedFx-Frontend-v2/blob/5a6bb3466292d8886048dfc48677679b96e2e386/src/components/Integrations/QLC/DialogAddEventListener.tsx#L67-L70


So either we update the frontend, or we keep the same API and maps names -> IDs somewhere else